src/main/java/org/cactoos/iterable/Shuffled.java

```diff
@@ -45,14 +48,26 @@ public Shuffled(final T... src) {
 
     /**
      * Ctor.
+     * @param rnd Randomizer.
      * @param src The underlying iterable
      */
-    public Shuffled(final Iterable<T> src) {
+    public Shuffled(final Random rnd, final Iterable<T> src) {
         super(
             new IterableOf<>(
-                () -> new org.cactoos.iterator.Shuffled<>(src.iterator())
+                () -> new org.cactoos.iterator.Shuffled<>(
+                    rnd,
+                    src.iterator()
+                )
             )
         );
     }
 
+    /**
+     * Ctor.
+     * @param src The underlying iterable
+     */
+    public Shuffled(final Iterable<T> src) {
+        this(new SecureRandom(), src);
```

Comment:
@andreoss let's have the primary constructor last in the class, so let's move this constructor before the other one